Dispatcher/Models/ArticleResponse.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  ArticleResponse.swift
+//  Dispatcher
+//
+//  Created by Ofek Cohen on 01/05/2023.
+//
+
+import Foundation
+
+struct ArticleResponse: Decodable {
+    let title: String
+    let author: String
+    let publishedDate: String
+    let publishedDatePrecision: String
+    let link: String
+    let cleanUrl: String
+    let excerpt: String?
+    let summary: String
+    let rights: String
+    let rank: Int
+    let topic: String
+    let country: String
+    let language: String?
+    let authors: String
+    let media: String
+    let isOpinion: Bool
+    let twitterAccount: String?
```

Comment:
Do you use all of the properties from the server in your app? Clean your objects by removing unnecessary values.